ecommerce_api_client/client.py

```diff
@@ -21,8 +21,8 @@ def __init__(self, url, signing_key=None, username=None, email=None, timeout=5,
 
         if oauth_access_token:
             auth = BearerAuth(oauth_access_token)
-        elif signing_key and username and email:
-            auth = JwtAuth(username, email, signing_key, tracking_context)
+        elif signing_key and username and email and full_name:
```

Comment:
The presence of a `signing_key` and `username` should be the only requirements gating use of JWT authentication. A valid JWT payload can be constructed even if `email` and `full_name` are `None`.